src/plugins/intel_cpu/src/nodes/fullyconnected.cpp

```diff
@@ -668,6 +741,156 @@ InferenceEngine::Precision FullyConnected::getRuntimePrecision() const {
+    dnnl::memory::dims normalizedInDims;
+    if (inDims.size() == 3) {
+        normalizedInDims = {inDims[0], inDims[2], dnnl::memory::dim{1}, inDims[1]};
+    } else if (inDims.size() == 2) {
+        normalizedInDims = {dnnl::memory::dim{1}, inDims[1], dnnl::memory::dim{1}, inDims[0]};
+    }
+    auto convInDesc = dnnl::memory::desc(normalizedInDims, inputDesc.data_type(), memory::format_tag::nhwc);
+
+    // make a fake shape: N, OC, 1, W
+    auto outDims = outputDesc.dims();
+    dnnl::memory::dims normalizedOutDims;
+    if (outDims.size() == 3) {
+        normalizedOutDims = { outDims[0], outDims[2], dnnl::memory::dim{1}, outDims[1]};
+    } else if (outDims.size() == 2) {
+        normalizedOutDims = { dnnl::memory::dim{1}, outDims[1], dnnl::memory::dim{1}, outDims[0]};
+    }
+    auto convOutDesc = dnnl::memory::desc(normalizedOutDims, outputDesc.data_type(), memory::format_tag::nhwc);
+
+    // make a fake shape: OC, IC, 1, 1
+    auto weightDims = getInputShapeAtPort(WEIGHTS_ID).getStaticDims();
+    dnnl::memory::dims normalizedWeightDims;
+    normalizedWeightDims = {static_cast<dnnl::memory::dim>(weightDims[0]),
+                            static_cast<dnnl::memory::dim>(weightDims[1]),
+                            dnnl::memory::dim{1},
+                            dnnl::memory::dim{1}};
+    auto convWeightDescAny = dnnl::memory::desc(normalizedWeightDims, wdt, dnnl::memory::format_tag::any);
+
+    std::shared_ptr<dnnl::convolution_forward::desc> desc;
+    if (withBiases) {
+        desc = std::make_shared<dnnl::convolution_forward::desc>(prop_kind::forward_scoring, dnnl::algorithm::convolution_direct,
+                                convInDesc, convWeightDescAny, biasDesc, convOutDesc,
+                                dnnl::memory::dims{1, 1},   // stride
+                                dnnl::memory::dims{0, 0},   // dilation
+                                dnnl::memory::dims{0, 0},   // paddingL
+                                dnnl::memory::dims{0, 0});  // paddingR
+    } else {
+        desc = std::make_shared<dnnl::convolution_forward::desc>(prop_kind::forward_scoring, dnnl::algorithm::convolution_direct,
+                                convInDesc, convWeightDescAny, convOutDesc,
+                                dnnl::memory::dims{1, 1},   // stride
+                                dnnl::memory::dims{0, 0},   // dilation
+                                dnnl::memory::dims{0, 0},   // paddingL
+                                dnnl::memory::dims{0, 0});  // paddingR
+    }
+
+    return DnnlDesriptor(desc);
+}
+
+bool FullyConnected::canBeExecutedInConv1x1() const {
+    bool shouldTry = false;
+    const auto inRank = getInputShapeAtPort(DATA_ID).getMaxDims().size();
+    const auto weightRank = getInputShapeAtPort(WEIGHTS_ID).getStaticDims().size();
+    // disable rank=4: rank == 4 means a matrix: N * IC * H * W --> N * (IC*H*W) --> N * IC * 1 * 1, it should not be efficient which acts as vector multiply
```

Comment:
```suggestion
    // disable rank=4: rank == 4 means a matrix: N * IC * H * W --> N * (IC*H*W) --> N * IC * 1 * 1, it should not be efficient since acts as a vector multiply
```

Reply:
Updated.